packages/re-windows.1.7.1/opam

```diff
@@ -6,4 +6,8 @@ build: [
   ["env" "OCAMLFIND_TOOLCHAIN=windows" "ocaml" "setup.ml" "-install"]
 ]
 remove: [["ocamlfind" "-toolchain" "windows" "remove" "re"]]
-depends: ["ocaml-windows" "ocamlfind"]
```

Comment:
Can be shortened to `depends: ["ocaml-windows" "ocamlbuild" {build}]` since `ocaml-windows` implies `ocamlfind`.

Reply:
Word.